runninglocally.md

```diff
@@ -0,0 +1,11 @@
+# Running Locally
+
+There come times when you want to run your code locally on your machine. While every repo is set up differently, there are a few trends across most repos that you can fall back on. 
+
+Most repos have a `README.md`, which may have the information you're looking for. If the README does not tell you what to do, don't fret! These steps below should work for most repos:
+
+1. Check if there's a `Makefile` in the repo. If there's no Makefile, do not continue to further steps and instead ask a fellow engineer. 
```

Comment:
This pattern is mainly used in golang repos. For frontend the pattern is more along the lines:

1. `nvm use 12` (we're still migrating to node 12 IIRC, so the version will be dictated by the `Makefile`
2. `npm install`
2. `ark start -l`